### ISSUE TITLE
Is this a broken test?

### ISSUE THREAD

**Issue by shreyasbapat** [poliastro/poliastro#507]
🐞 **Problem**
This test(which is parametrized) doesn't really seem to be parametrized. 
```
@pytest.mark.parametrize("value", [1 * u.h, 10 * u.deg])
def test_orbit_propagate_retains_plane(value):
    r = [1E+09, -4E+09, -1E+09] * u.km
    v = [5E+00, -1E+01, -4E+00] * u.km / u.s

    ss = Orbit.from_vectors(Sun, r, v, plane=Planes.EARTH_ECLIPTIC)

    orig_frame = ss.frame

    final_ss = ss.propagate(1 * u.h)
    expected_frame = orig_frame.replicate_without_data(obstime=final_ss.epoch)

    assert final_ss.frame.is_equivalent_frame(expected_frame)
```

🖥 **Please paste the output of following commands**

* `conda info -a` (only if you have conda)
* `conda list` (only if you have conda)
* `pip freeze`

<details>
  
```bash
# Paste your output here:

```
</details>

* `python -c "import poliastro.testing; poliastro.testing.test()"`

<details>

```bash
# Paste your output here:

```
</details>

🎯 **Goal**

Maybe change it? 
💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
You refer to this test (use the "Copy permalink" option):

https://github.com/poliastro/poliastro/blob/72eff1bc42d4c5f1b3b58caab40367b0c45639e9/src/poliastro/tests/tests_twobody/test_orbit.py#L333-L345

Using the "Blame" view on GitHub I found that it was introduced in this commit:

https://github.com/poliastro/poliastro/commit/a09024470e218ae3c32c8615b92728117bffa8fd#diff-7c169630b1f7e9c70aaba5dfa9d0bd43R323

And it's clearly a typo, perhaps due to copy-paste. The parametrization should be removed. Good catch!

**Comment by shreyasbapat** [comment]
> use the "Copy permalink" option

Will keep in mind :)

**Comment by Juanlu001** [comment]
Fixed in #637
